gradle/libs.versions.toml

```diff
@@ -1,11 +1,12 @@
 [versions]
-kotlin = "1.7.10"
-kotlinx-coroutines = "1.6.1"
-kotlinx-serialization = "1.3.0"
-androidxComposeCompiler = "1.3.0"
+kotlin = "1.7.20"
+kotlinx-coroutines = "1.6.4"
+kotlinx-serialization = "1.4.0"
```

Comment:
```suggestion
kotlinx-serialization = "1.4.1"
```

Reply:
https://github.com/Kotlin/kotlinx.serialization/releases/tag/v1.4.1

Reply:
Thanks! I actually had that locally along with the Zipline bump but forgot to amend my commit before pushing.